.golangci.yml

```
@@ -32,6 +36,8 @@ run:
     - web_src
 
 linters-settings:
+  nakedret:
+    max-func-lines: 5 
```

I think 5 is harsh for a Go program:
You already have 3 with a single `if _, err := someFunc(); err != nil`...
What about 10 instead?
```suggestion
    max-func-lines: 10
```

---

Well, personally I don't want any `nakedret` at all so I understand this is controversial. 
Lets see what other people want.

---

I don't like naked return either 😅

---

I also agree to forbid the naked returns in large functions.

At the moment, max-func-lines: 5 or 10 lines are both fine to me. Forbidding all is also fine to me.

---

I am fine with all values, I don't have a preference (yet?).

---

Ok, I've just disabled nakedreturns for the linter and all the ones that existed are fixed.